recipes/py2vega/meta.yaml

```
@@ -0,0 +1,38 @@
+{% set name = "py2vega" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 370d27e12d4add32a6b6092c347905d716d18687589bc59c0b90b76fdf79619b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<35]
```

I think this recipe can be ``noarch``
```suggestion
  noarch: python
```

and after that you can select the python versions which you want to